Makefile

```diff
@@ -48,6 +73,15 @@ ${BOARD}.img: boot/boot-${BOARD}.scr $(ROOTFS)
 	rm -f $@
 	tar -Szcf $@ $<
 
+## Convenience, avoiding the need to repeat yourself about which board
+img: $(BOARD).img
+tar: $(BOARD).tar
+tgz: $(BOARD).tar.gz
+
+dd: $(BOARD).img
+	sudo dd if=$(BOARD).img of=/dev/mmcblk0 bs=4096
```

Comment:
I think it would be safest to remove this too. The user can always write their own script if they want to do this a lot, or we could echo the suggested command.